src/patternfly/components/CodeEditor/code-editor.scss

```diff
@@ -0,0 +1,91 @@
+.pf-c-code-editor {
+  // control
+  --pf-c-code-editor__control--Height: calc(var(--pf-global--spacer--form-element) + var(--pf-global--spacer--form-element) + var(--pf-global--FontSize--md) * var(--pf-global--LineHeight--md) - var(--pf-global--BorderWidth--sm));
+
+  // action group
+  --pf-c-code-editor__action-group--ZIndex: var(--pf-global--ZIndex--xs);
+  --pf-c-code-editor__action-group--c-button--m-control--Color: var(--pf-global--Color--200);
+  --pf-c-code-editor__action-group--c-button--m-control--hover--Color: var(--pf-global--Color--100);
+  --pf-c-code-editor__action-group--c-button--m-control--focus--Color: var(--pf-global--Color--100);
+
+  // code
+  --pf-c-code-editor__code--before--BorderColor: var(--pf-global--BorderColor--100);
+  --pf-c-code-editor__code--before--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-code-editor__code--BackgroundColor: var(--pf-global--BackgroundColor--100);
+
+  // tab
+  --pf-c-code-editor__tab--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-code-editor__tab--Color: var(--pf-global--Color--200);
+  --pf-c-code-editor__tab--ZIndex: var(--pf-global--ZIndex--xs);
+  --pf-c-code-editor__tab--Top: var(--pf-c-code-editor__code--before--BorderWidth);
+  --pf-c-code-editor__tab--PaddingTop: var(--pf-global--spacer--form-element);
+  --pf-c-code-editor__tab--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-code-editor__tab--PaddingBottom: var(--pf-global--spacer--form-element);
+  --pf-c-code-editor__tab--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-code-editor__tab--before--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-code-editor__tab--before--BorderTopColor: var(--pf-global--BorderColor--100);
+  --pf-c-code-editor__tab--before--BorderRightColor: var(--pf-global--BorderColor--100);
+  --pf-c-code-editor__tab--before--BorderBottomColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-code-editor__tab--before--BorderLeftColor: var(--pf-global--BorderColor--100);
+
+  width: 100%;
+}
+
+.pf-c-code-editor__control {
```

Comment:
I'm not sure about the name `__control` since it's a container for the stuff at the top, including the tab. It might be better labeled `__header` maybe? Then you could rename `__action-group` to `__control` or `__controls`.

Reply:
ok that sounds good to me